src/Database/Mysql/Driver/Mysql.php

```diff
@@ -63,7 +63,8 @@ public function update(Tenant $tenant): bool
         $tempTenant->{$tempTenant->getTenantKeyName()} = $tenant->getOriginal($tenant->getTenantKeyName());
         $originalTenant = Tenancy::getTenant();
 
-        Tenancy::setTenant($tempTenant);
+        $resolver = resolve(\Tenancy\Affects\Connection\ConnectionResolver::class);
+        $resolver->__invoke($tempTenant, Tenancy::getTenantConnectionName());
```

Comment:
Looks like this method needs some cleaning up. Move setting and retrieving the connection to a new method?

Reply:
Have been thinking about moving the functionality of grabbing the tables from:
Using a tenant to basically query SHOW TABLES for us.
To:
Using the system connection to grab the tables using a raw query SHOW TABLES.

In that way, the code won't do any weird fiddling in order to grab the tables.